test/bumbler_test.rb

```diff
@@ -55,15 +55,15 @@ def bumbler(command = "", **args)
 
     it "works with non-rails project" do
       write("Gemfile", "source 'https://rubygems.org'")
-      bumbler("").must_equal "0 of 0 gems required\nSlow requires:\n"
+      bumbler("").must_include "0 of 0 gems required\nSlow requires:\n"
```

Comment:
bundler spits out some extra output for me. at first I thought these were failing on master, but perhaps something is different with my local setup. I made them less strict but can figure something else out if you don't like this

Reply:
ideally ignore the known bad output since `equal` will always show new warnings